src/date.ml

```diff
@@ -584,8 +584,8 @@ value string_slash_of_date conf date =
         " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 1) ^ ")"
   | Dgreg d Dfrench ->
       let d1 = Calendar.french_of_gregorian d in
-      slashify_dmy (translate_dmy conf (decode_dmy conf d1) Dfrench True) d1 ^
-        " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 2) ^ ")"
+      slashify_dmy (translate_dmy conf (decode_dmy conf d1) Dfrench True) d1(* ^
+        " (" ^ (transl_nth conf "gregorian/julian/french/hebrew" 2) ^ ")" *)
```

Comment:
Isn't it useful for no genealogist people ? Also why keeping the other then ? hebrew should be treated the same way ?